### DISCUSSION TITLE
docs/intro

### DISCUSSION THREAD

**Discussion by giscus** [wjwei-handsome/wjwei-docusaurus#5] [General]
# docs/intro

分享一些个人学习过程中的粗略理解，不足之处请批评指正。

https://www.wjwei.blog/docs/intro

**Comment by wjwei-handsome** [comment]
测试一下
`测试一下`
**测试一下***

**Comment by Lingerssss** [comment]
你这个主页不错，能分享下图标怎么搞的吗

**Reply by wjwei-handsome** [reply]
感谢关注！这一部分的源码在此处：https://github.com/wjwei-handsome/wjwei-docusaurus/tree/main/src/components/GridList ，包含了一个组件和对应的样式文件，至于图标来源，都是网上找的统一格式512*512的svg。
